templates/_components/quote_and_image.html.twig

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row govuk-!-padding-9">
+    <div class="govuk-grid-column-one-quarter govuk-heading-xl govuk-!-padding-left-5 quote">“</div>
+    <div class="govuk-grid-column-one-third quote_image_width">
+        <div class="govuk-hint govuk-!-font-size-24">
+        {{ component.quote_and_image_quote_and_image_quote | striptags | raw }}
+        </div>
+        <div class="govuk-body govuk-!-font-size-20 govuk-!-font-weight-bold govuk-!-margin-0">
+        {{ component.quote_and_image_quote_and_image_name | striptags | raw }}
+        </div>
+        <div class="govuk-body govuk-!-font-size-20">
+        {{ component.quote_and_image_quote_and_image_job | striptags | raw }}
+        </div>
+    </div>
+    <div class="govuk-grid-column-one-quarter govuk-!-width-one-quarter quote_image">
+        {% if component.quote_and_image_quote_and_image_image %}
+            <figure class="image-wrapper">
+                <picture class="">
+                    <source component="(max-width: 768px)" srcset="{{ component.quote_and_image_quote_and_image_image.byName('medium_large') }}">
+                    <source component="(min-width: 1024px)" srcset="{{ component.quote_and_image_quote_and_image_image.byName('large') }}">
+                    <source component="(min-width: 1920px)" srcset="{{ component.quote_and_image_quote_and_image_image.byName('full') }}">

```

Comment:
Ok, so I've spotted a syntax error here, and also did some prototyping to check how this works (I need to read up more on the picture element myself).

First off, `component` should be `media`.

Secondly, the order that these go in is important.

Browsers essentially evaluate these top to bottom, and as soon as they hit a matching media query, they skip the rest.

This means that at present the last source (min-width: 1920px) will never get used.

I've setup two simple examples to illustrate this below.

Correct (order reversed):
https://jsfiddle.net/5xu84dzq/1/

Incorrect (as it is currently implemented):
https://jsfiddle.net/a5w082n9/

(If you resize the screen you should see what I mean).

There might also be other instances like this in pre-existing code from us ... but I wouldn't worry about that much right now.

Lastly, there is a blindspot in these media queries between 768px and 1024px. You can see this in both the examples I set as well.

In this blindspot, the fallback image will be used. This may be desired or not - so I'll leave it up to you on whether you fix this or not.